src/components/addComment.jsx

```diff
@@ -0,0 +1,79 @@
+import { useState } from "react";
+
+const AddComment = ({
+    newBody,
+    setNewBody,
+    user,
+    postComment,
+    review_id,
+    setComments,
+}) => {
+    const [commentFailed, setCommentFailed] = useState(false);
+    const [newCommentLoading, setNewCommentLoading] = useState(false);
+
+    const submitComment = (event) => {
+        event.preventDefault();
+        setNewBody(newBody.trim());
+        if (newBody.length === 0) {
+            return;
+        }
+        const commentToAdd = { username: user.username, body: newBody };
+        setNewCommentLoading(true);
+        postComment(review_id, commentToAdd)
+            .then((response) => {
+                setNewCommentLoading(false);
+                setComments((currComments) => {
+                    return [response, ...currComments];
+                });
+            })
+            .catch((err) => {
+                console.log("im in the catch");
+                //<p>{`Error: ${err}`}</p>;
+                setNewCommentLoading(false);
+                setCommentFailed(true);
+                // setNewBody("");
+            });
+    };
+
+    if (commentFailed) {
+        return (
+            <div>
+                <h3>Error: Your comment could not be posted</h3>
+            </div>
+        );
+    }
+
+    if (newCommentLoading) {
+        return (
+            <div>
+                <h3>Adding your comment...</h3>
+            </div>
+        );
+    }
+
+    return (
+        <form onSubmit={submitComment}>
+            <label>{commentFailed.toString()}</label>
```

Comment:
<img width="542" alt="image" src="https://user-images.githubusercontent.com/72257311/233399900-2e37dce2-7e8b-47a0-85b0-4a0242435b89.png">
This is rendering on the page as "false"